### ISSUE TITLE
Pointer errors in windows numpy.core (migrated from Trac #145)

### ISSUE THREAD

**Issue by thouis** [thouis/numpy-trac-migration#458]
Original ticket http://projects.scipy.org/numpy/ticket/145
Reported 2006-06-15 by trac user gregas, assigned to @thouis not really:pearu.

I appologize for the vauguess of this since I'm not at all expert with the numpy internals. Still, I did not see this reported and it sure is annoying.

While running within the ActiveState Komodo IDE, I have the following problem:

I run:

```
from numpy.core import *
print "foo"
```

If I place a breakpoint at line 2, I get a window popup saying the memory at 0x00000014 cannot be "read". This crashes the program being tested (but not the IDE), but does not allow breakpoint debugging. I also get a similiar memory error from

```
from numpy.core import *
foo()
```

Where foo() is undefined.

This behaviour exists in v 0.94-0.98, but not in v 0.92.

I would be happy to help with more information if somebody w/ more experience wants to ask specific questions.

I don't normally hang around here, but I can be reached at gregasmaximus@XXXgmail.com

Thanks to all for their hard work on numpy and scipy.

Greg

**Comment by thouis** [comment]
Comment in Trac by trac user gregas, 2006-06-15

Note: This may be obvious, but the code examples that I gave above were originally two lines. Somehow when it got posted they were concatonated

**Comment by thouis** [comment]
Comment in Trac by @XXXthouis not really:brickZA, 2006-06-16

gregas: You should surround code with

```
code
```

since wiki-formating is used. See the nice help link above the edit box :)

**Comment by thouis** [comment]
Comment in Trac by @XXXthouis not really:brickZA, 2006-06-16

Argh, servers me right for not previewing. What you actually need to do is

```
     code
```

since wiki-formating is used. See the nice help link above the edit box :)

**Comment by thouis** [comment]
Comment in Trac by @XXXthouis not really:teoliphant, 2006-06-26

Need more information to do anything about this.

**Comment by thouis** [comment]
Comment in Trac by @XXXthouis not really:dmcooke, 2006-07-17

fixed formatting in description.

**Comment by thouis** [comment]
Comment in Trac by @XXXthouis not really:teoliphant, 2006-07-17

Because of when the error was first reported, I strongly suspect that this is due to the interaction between the PackageLoader class and the magic that Komodo is using for self inspection.   I'm not sure where the error lies, exactly, however.  It appears that Komodo is making an incorrect assumption about the objects it's inspecting.  

Perhaps a fix for the user is to make sure that numpy.core is actually loaded before Komodo plays it's games with from numpy.core import *

In other words, use:

```
import numpy.core
from numpy.core import *
```

**Comment by thouis** [comment]
Comment in Trac by @XXXthouis not really:teoliphant, 2006-07-21

This should be fixed because we don't use package-loader anymore in numpy's **init**.py
